src/Microsoft.Health.Dicom.Core/Features/Common/IFileStore.cs

```diff
@@ -80,4 +80,14 @@ public interface IFileStore
     Task DeleteOldFileIfExistsAsync(
         VersionedInstanceIdentifier versionedInstanceIdentifier,
         CancellationToken cancellationToken = default);
+
+    /// <summary>
+    /// Asynchronously gets a streaming file from the file store.
+    /// </summary>
+    /// <param name="versionedInstanceIdentifier">The DICOM identifier.</param>
+    /// <param name="cancellationToken">The cancellation token.</param>
+    /// <returns>A task that represents the asynchronous get operation.</returns>
+    Task<Stream> GetStreamingFileAsync(
```

Comment:
Should we double-down on the "file" aspect and make this a method like `OpenReadAsync`?

Reply:
Probably. Right now kept it in the same pattern as the rest of the method names